app/src/main/java/com/example/foody/recipe_search/presentation_mvvm/ui/SearchScreen.kt

```diff
@@ -59,20 +65,21 @@ fun SearchScreen(
     viewModel: SearchViewModel = hiltViewModel()
 ) {
     val state by viewModel.state.collectAsState()
-
     val performSearch: () -> Unit = remember { { viewModel.search() } }
     val onValueChanged: (String) -> Unit = remember { { viewModel.updateSearchTerm(it) } }
     val navigateWith: (String) -> Unit =
         remember { { viewModel.navigateTo(SearchNavigationEvent.ToDetails(it)) } }
+    val onFabClick: () -> Unit = remember { { viewModel.flipSearchBarExpandedState() } }
```

Comment:
would this work: `remember { viewModel::flipSearchBarExpandedState }`

Reply:
:) i'll try...

Reply:
When I do that it says: "The expression is unused".

Reply:
unused or "suspicious"? I think it should say suspicious, because it is "wrapped" in curly braces {}, and compiler knows that is how a lambda is defined, so it is asking you "are you sure you did not make a mistake", as in started to create a lambda as in `{ someCall(it) }` but made a mistake to do `{ ::someCall }`, when it should be just `::someCall` without curly braces. In our case, there is no mistake, since curly braces are for `remember` and not for the lambda.

Reply:
Both... depends how you get the info. Ok, I'll try again.

Reply:
That is why it is not a warning, but just a "weak warning" or "info" (grey, not yellow) - as compiler cannot know for sure, but just want's to give you info so you can pay attention. Since we know it is fine, we can safely annotate the `SearchScreen` fun with `@Suppress("SuspiciousCallableReferenceInLambda")`

Reply:
Applied in the next PR